README.rst

```diff
@@ -9,6 +9,7 @@ tidalapi
 
 Unofficial Python API for TIDAL music streaming service.
 
+Requires Python 3.7 or higher.
```

Comment:
This might change very soon.. We should remember to update it to the latest supported version.